src/js/game/LevelMVC/LevelView.js

```diff
@@ -2015,6 +2025,102 @@ module.exports = class LevelView {
     return sprite;
   }
 
+  determineRedstoneSprite(x, y, blockType, levelData) {
+        let foundAbove = false;
+        let foundBelow = false;
+        let foundRight = false;
+        let foundLeft = false;
+        let belowIndex = (this.yToIndex(y + 1)) + x;
+        let aboveIndex = (this.yToIndex(y - 1)) + x;
+        let leftIndex = (this.yToIndex(y)) + x - 1;
+        let rightIndex = (this.yToIndex(y)) + x + 1;
+
+        let borderCount = 0;
+
+        //need to ensure these are in bounds
+        if (y === levelData.planeHeight) {
```

Comment:
Hmm, you're right: it may indeed make more sense for the model to store "redstone_wire" everywhere, and the view to determine how to render it.  Let me think about this a little more.